samples/wifi/raw_tx_packet/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(nrf_wifi_raw_tx_packet)
 
 target_include_directories(app PUBLIC ${ZEPHYR_BASE}/subsys/net/ip)
+target_include_directories(
+	app
+	PRIVATE
+	${ZEPHYR_BASE}/../nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/)
```

Comment:
Please limit till `nrfxlib`, and use `<nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h>`